gems/aws-sdk-core/aws-sdk-core.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency('jmespath', '~> 1.0')
   spec.add_dependency('aws-partitions', '~> 1.0')
   spec.add_dependency('aws-sigv4', '~> 1.0') # necessary for making Aws::STS API calls
-  spec.add_dependency('aws-eventstream', '~> 0.0.0')
+  spec.add_dependency('aws-eventstream', '~> 1.0') # necessary for binary eventstream
```

Comment:
This is insufficient for launch, as we would also need to ensure that service gems which require the presence of eventstream to have a minimum bound on their dependency on `aws-sdk-core`, otherwise bundler will accept invalid dependency graphs.